src/main/java/core/basesyntax/service/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.time.LocalDateTime;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    private final Path path;
+
+    public FileWriterServiceImpl(Path path) {
+        this.path = path;
+    }
+
+    @Override
+    public void writeReportToFile(String report) {
+        Path fileName =
+                Path.of(path.toString()
+                        + File.separator
+                        + "report-"
+                        + LocalDateTime.now()
+                        + ".csv");
+        try {
+            Files.createFile(fileName);
+            Files.write(fileName,report.getBytes(), StandardOpenOption.APPEND);
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html

read about method `write`, line 27 is redundant